dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferBasics.cs

```diff
@@ -31,14 +45,18 @@
 PressEnter();
 
 // Upload a local directory to an S3 bucket.
-var keyPrefix = "UploadFolder";
+DisplayTitle("Upload all files from a local directory");
```

Comment:
Missing punctuation?

Reply:
If this text functions as a title, no ending punctuation is required.

Reply:
This is a title.